instagram-app/src/App.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import data from './dummy-data';
+// Global Styles
+import { GlobalStyle } from './components/styles/Global';
+// Components
+import SearchBar from './components/SearchBar/SearchBar';
+import PostContainer from './components/PostContainer/PostContainer';
+
+class App extends Component {
+  constructor( props ) {
```

Comment:
Quick little tip, when using `create-react-app` you no longer need to wrap your state in a `constructor` and call `super()` 

So you can take this
```
 constructor( props ) {
    super( props );
    this.state = {
      posts: [],
      nextCommentId: 0,
    };
```

and convert it to this
```
state = {
   posts: [],
   nextCommentId: 0,
};
```

Right off the bat you can see this is already much more concise and easier to read. The reason you can do this is cause of tool named [babel](https://babeljs.io/). Babel is a transpiler, which means it takes the current code you've written and rewrites it into the same language typically to a different syntax/version that's well supported on different browsers/environments. So when your state isn't wrapped in a constructor and calling super babel automatically adds that code in there so you don't have to.

for example if you write
```
const foo = () => {
  return 'bar'
}
```

babel with transpiles that into
```
var foo = function foo() {
    return 'bar';
}
```

you can see it has the exact same functionality but just a different syntax that is well supported on all web browsers

Reply:
Thanks for the feedback.  Abstracting State away from the constructor method does clean up the code better, knowing that babel will compile the code to be correct.  I will institute this on the next commit.

Thanks,

Donald Whitely

Reply:
Awesome! 🙌 Can't wait to see it!